tests/nnfw_api/src/GenModelTest.h

```diff
@@ -228,6 +228,10 @@ class GenModelTestContext
       {
         _backends.push_back(backend);
       }
+      else if (backend == "ruy")
```

Comment:
`if (backend == "cpu" || if (backend == "ruy"))` looks better. I do like this on #5021

Reply:
@YongseopKim Thanks! It looks better. I've updated.